config/database.yml.sample

```diff
@@ -1,29 +0,0 @@
-# SQLite version 3.x
```

Comment:
Leave this file. It will sample for those people that will maintain this project later.